pkgs/README.md

```diff
@@ -23,7 +23,18 @@ See the [CONTRIBUTING.md](../CONTRIBUTING.md) document for more general informat
 
 ## Quick Start to Adding a Package
 
-To add a package to Nixpkgs:
+Each new package has a cost for the maintainers, Hydra and users downloading
+Nixpkgs. Before adding a new package, please consider the following questions:
+* Does the package have a release?
+* How realistic is it that it will be used by other people?
+* Are you going to maintain the package?
```

Comment:
Wondering if we should specify a timeframe here, like

```suggestion
* Are you going to keep maintaining the package at least 6 months from today?
```

Reply:
It's a good question, I don't know.

On the surface, it seems reasonable. But it also opens up new questions like what is the appropriate duration, is there a single duration that works for all packages, and what happens if they don't respect that commitment?

Or we ask the contributor to make a good-faith commitment and leave it unspecified.

Reply:
A more reasonable timeframe would be the next NixOS release cycle, but I'm fine with a good faith commitment as well.

Reply:
It would be nice to move towards that for sure. I wonder what existing maintainers would think of this. Keeping this open for now to let it simmer a bit.

Reply:
Added some wording to make the minimum duration normative but not enforced: a0378d09042d3cc4ed5166d20c7f0f9f27913511